enterprise-suite-latest/Makefile

```diff
@@ -23,6 +23,8 @@ include ../$(ES_CHART)/Makefile
 CHART_DIR = ../$(ES_CHART)
 COMPONENTS := $(wildcard ../$(ES_CHART)/*/.)
 SUBCOMPONENTS := $(wildcard ../$(ES_CHART)/*/*/.)
+# Generate dev helm settings from enterprise-suite settings and values-latest.yaml file
+DEV_HELM_SETTINGS := $(DEV_HELM_SETTINGS),$(shell yq r -j values-latest.yaml | sed -e 's/[}{"]//g' -e 's/:/=/g')
```

Comment:
Why not just use `--set-file values-latest.yaml` ?

Reply:
Uhhh...  Cuz it never occurred to me for some reason...  Durn.  That's a much better solution.

Reply:
I use it for minishift 
helm install --name=es-local-latest -f enterprise-suite-latest/values-latest.yaml  --debug --set minikube=true  enterprise-suite

Reply:
Went with `--values` (same as `-f`).  Interestingly, I find that the `--set-file` flag is gone from `helm install` as of v 2.9.1 at least.  (Unfortunate that it's still documented at https://docs.helm.sh/helm/#helm-install.  But when I read the doco carefully, I see it wasn't right for the job anyway.  I've misunderstood that flag since forever...)

Reply:
Aside:  Upgraded to helm 2.10.0 and the `--set-file` flag is there now.  Still not the right thing for this job but the mystery is solved.  :-)